common/services/user-locations.test.js

```diff
@@ -1,4 +1,5 @@
 const proxyquire = require('proxyquire')
+const { encodeToken } = require('../../test/unit/common-helpers')
```

Comment:
Maybe just `helpers` instead of `common-helpers`?

Reply:
have removed this as its now only being used in one file